consensus/src/chained_bft/block_storage/block_tree.rs

```diff
@@ -177,7 +156,6 @@ where
     fn remove_block(&mut self, block_id: HashValue) {
         // Remove the block from the store
         self.id_to_block.remove(&block_id);
-        self.id_to_votes.remove(&block_id);
```

Comment:
this needs to be updated with `PendingVotes` to avoid leaks

Reply:
I don't think we need to explicitly clean things in PendingVotes when a block is pruned. As I stated in the PendingVotes, we need to fully clear it up when a new epoch starts. During the epoch the PendingVotes is keeping the latest votes per author, hence no explicit pruning is required.